docker/generic/start_proxy.py

```diff
@@ -529,46 +574,17 @@ def make_argparser():
         SSL protocols (e.g., --ssl_protocols=TLSv1.1 --ssl_protocols=TLSv1.2).
         ''')
 
-    parser.add_argument(
-        '--transcoding_always_print_primitive_fields',
-        action='store_true', help='''Whether to always print primitive fields
-        for grpc-json transcoding. By default primitive fields with default
-        values will be omitted in JSON output. For example, an int32 field set
-        to 0 will be omitted. Setting this flag to true will override the
-        default behavior and print primitive fields regardless of their values.
-        Defaults to false
-        ''')
-
-    parser.add_argument(
-        '--transcoding_always_print_enums_as_ints', action='store_true',
-        help='''Whether to always print enums as ints for grpc-json transcoding.
-        By default they are rendered as strings. Defaults to false.''')
-
-    parser.add_argument(
-        '--transcoding_preserve_proto_field_names', action='store_true',
-        help='''Whether to preserve proto field names for grpc-json transcoding.
-        By default protobuf will generate JSON field names using the json_name
-        option, or lower camel case, in that order. Setting this flag will
-        preserve the original field names. Defaults to false''')
-
-    parser.add_argument(
-        '--transcoding_ignore_query_parameters', action=None,
-        help='''
-         A list of query parameters(separated by comma) to be ignored for
-         transcoding method mapping in grpc-json transcoding. By default, the
-         transcoder filter will not transcode a request if there are any
-         unknown/invalid query parameters.
-         ''')
-
-    parser.add_argument(
-        '--transcoding_ignore_unknown_query_parameters', action='store_true',
-        help='''
-        Whether to ignore query parameters that cannot be mapped to a
-        corresponding protobuf field in grpc-json transcoding. Use this if you
-        cannot control the query parameters and do not know them beforehand.
-        Otherwise use ignored_query_parameters. Defaults to false.
-        ''')
+    parser.add_argument('--enable_grpc_backend_ssl',
+        action='store_true', help='''
+        This flag added for backward compatible for ESPv1 and will be deprecated.
+        Enable SSL for gRPC backend. ESPv2 auto enables SSL if schema `grpcs` is
+        detected.''')
 
+    parser.add_argument('--grpc_backend_ssl_root_certs_file',
```

Comment:
are we going to support 

grpc_backend_ssl_private_key_file
grpc_backend_ssl_cert_chain_file

Reply:
yes,  will follow up in next PR.